scrapy/utils/spider.py

```diff
@@ -4,15 +4,18 @@
 from scrapy.spiders import Spider
 from scrapy.utils.defer import deferred_from_coro
 from scrapy.utils.misc import arg_to_iter
-try:
-    from scrapy.utils.py36 import collect_asyncgen
-except SyntaxError:
-    collect_asyncgen = None
 
 
 logger = logging.getLogger(__name__)
 
 
+async def collect_asyncgen(result):
```

Comment:
I'm using this function in a test in the async start_requests PR, I've moved it to utils.asyncgen there.